_posts/2024-07-22-kubeflow-1.9-release.md

```diff
@@ -0,0 +1,226 @@
+
+### Manifests
+
+#### Installation and documentation improvements
+
+The [documentation](https://github.com/kubeflow/manifests?tab=readme-ov-file#upgrading-and-extending) has been improved and now contains guidelines for upgrading and extending the Kubeflow Platform for administrators. New users can now install kubeflow on their laptop in just a few minutes
+
+Platform dependencies updates:
+
+<table>
+  <tr>
+   <td><strong>Component</strong>
+   </td>
+   <td><a href="https://kubernetes.io/releases/">Kubernetes</a>
+   </td>
+   <td><a href="https://github.com/kubernetes-sigs/kustomize/releases">Kustomize</a>
+   </td>
+   <td><a href="https://istio.io/latest/news/releases/">Istio</a>
+   </td>
+   <td><a href="https://github.com/dexidp/dex/releases">Dex</a>
+   </td>
+   <td><a href="https://cert-manager.io/docs/installation/supported-releases/">Cert-Manager</a>
+   </td>
+   <td><a href="https://knative.dev/docs/reference/relnotes/">Knative</a>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>KF 1.9 Version</strong>
+   </td>
+   <td>1.27 - 1.29
+   </td>
+   <td>5.2.1+
+   </td>
+   <td>1.22.1
+   </td>
+   <td>2.39.1
+   </td>
+   <td>1.14.5
+   </td>
+   <td>1.12.4
+   </td>
+  </tr>
+</table>
+
+
+> **NOTE** Kubernetes 1.30+ is also expected to work, but was not officially tested.
+
+> **NOTE** Kustomize 5.2.1+ support with way less warnings. So platform engineers have a modern and supported installation tool chain.
+
+#### Integration with third-party ML tools
+
+We have updated the third-party components in /contrib to provide integration with the broader ML ecosystem. 
+
+- [Bentoml](https://github.com/kubeflow/manifests/tree/master/contrib/bentoml) 1.2.28 and 1.1.21
+- [Seldon](https://github.com/kubeflow/manifests/tree/master/contrib/seldon) 1.18.1
+- [Ray](https://github.com/kubeflow/manifests/tree/master/contrib/ray) 2.23 and Kuberay 1.1.1
+
+Find a detailed Manifests WG roadmap [here](https://github.com/kubeflow/manifests/issues/2592).
+
+## KServe
+
+We upgraded to [KServe 0.13](https://kserve.github.io/website/0.13/blog/articles/2024-05-15-KServe-0.13-release/) from the previous version 0.11. This release includes:
+
+- **Enhanced Hugging Face Runtime Support**: Hugging Face models are supported out-of-the-box, implementing a [KServe Hugging Face Serving Runtime](https://github.com/kserve/kserve/tree/master/python/huggingfaceserver). Currently supported tasks include sequence classification, token classification, fill mask, text generation, and text to text generation.
+- **vLLM Support**: Dedicated runtime support for [vLLM](https://docs.vllm.ai/en/latest/) is now included, streamlining the deployment process for LLMs.
+- **OpenAI Schema Integration**: KServe now supports endpoints for generative transformer models, following the OpenAI protocol.  This enables KServe to be used directly with OpenAI’s client libraries or third-party tools like LangChain and LlamaIndex.
+
+## Documentation
+
```

Comment:
The documentation for Kubeflow 1.9 starts here:  (need link for 1.9 page on Kubeflow.org)

Reply:
Added a link to the docs. I used the "main" branch link, not a specific v1.9 since we don't cut this until release and our versioning in the docs website is not a great experience